api/rpc/stat/stat.go

```diff
@@ -1,44 +1,239 @@
 package stat
 
 import (
-	"time"
-
+	"fmt"
 	"github.com/appcelerator/amp/data/influx"
 	"golang.org/x/net/context"
+	//"time"
 )
 
 //Stat structure to implement StatServer interface
 type Stat struct {
-	conn influx.Influx
+	Influx influx.Influx
+}
+
+//CPUQuery Extract CPU information according to StatRequest
+func (s *Stat) CPUQuery(ctx context.Context, req *StatRequest) (*CPUReply, error) {
+	idFieldName, nameFieldName := getIDNameFields(req)
+	fmt.Println(idFieldName, nameFieldName )
+	query := buildQueryString(req, idFieldName, "cpu")
+	res, err := s.Influx.Query(query)
+	if err != nil {
+		return nil, err
+	}
+
+	cpuReply := CPUReply{}
+	list := res.Results[0].Series[0].Values
+	cpuReply.Entries = make([]*CPUEntry, len(list))
+	for i, row := range list {
+		entry, err := s.ressourceCPUQuery(ctx, req, idFieldName, nameFieldName, row[1].(string))
```

Comment:
After test, LIMIT 1 doesn't work as excepted. we got just on row where we need to got all the row corresponding to all the container matching the filter